frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -94,9 +92,11 @@ const whitelistedFilterKeys = ['PID', 'PIN', 'STREET_ADDRESS_1', 'LOCATION'];
  * @param filter The map filter.
  */
 const getQueryParams = (filter: IPropertyFilter): IGeoSearchParams => {
+  // The map will search for either identifier.
+  const pinOrPidValue = filter.pinOrPid ? filter.pinOrPid?.replace(/-/g, '') : undefined;
   return {
-    PID: filter.pid ? filter.pid?.replace(/-/g, '') : undefined,
-    PIN: filter.pin ? filter.pin?.replace(/-/g, '') : undefined,
+    PID: pinOrPidValue,
+    PIN: pinOrPidValue,
     STREET_ADDRESS_1: filter.address,
```

Comment:
Did you test this with a live GeoServer? (dev container is fine)
I ask because there is a bit of change in the logic for the map here.

**Original:**
- Only one of PID or PIN is captured in the form so the query params would only have either PID=123 or PIN=123

**Updated:**
- We now send **two** query params with same value: `&PID=123&PIN=123`

This would likely affect the CQL query we send to GeoServer (@devinleighsmith  might be more familiar with that than I). See the code I'm talking about in `mapUtils.tsx` (lines 373-385)

```tsx
/**
 * Convert any object to a cql filter string, assuming the object's keys should be used as CQL filter properties.
 * AND all object keys together within the generated cql filter string.
 * @param object an object to convert to a cql filter string.
 */
export const toCqlFilter = (object: any) => {
  const cql: string[] = [];
  Object.keys(object).forEach((key: string) => {
    if (object[key]) {
      if (key === 'PID' && object[key]) {
        cql.push(`PIN ilike '%25${object[key]}%25' OR PID ilike '%25${object[key]}%25'`);
      } else {
        cql.push(`${key} ilike '%25${object[key]}%25'`);
      }
    }
  });
  return cql.length ? `cql_filter=${cql.join(' AND ')}` : '';
};

```

Reply:
Effectively we would be sending an additional "AND" CQL query to GeoServer (i.e. `http://geoserver/...?cql_filter="PID ilike 123 AND PIN ilike 123"`) which would likely return no results (unless there is a property with same partial PID and PIN at the same time

Reply:
Ah, good observation. Previously only the pin parameter would be filled since the filter was mapping the pin/din entry to pin only.

I tested it on my local machine and it seemed to work